### PR TITLE
add doc for npm install-test

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

**Comment by KenanY** [comment]
> Install and test a package

Hmm, is it just me or does this sound like this will install a package and then run its tests? The use-case is installing dependencies and then running `npm test`, no? Maybe "Install dependencies then test" or "Install dependencies and then run tests"?

**Comment by ashleygwilliams** [comment]
hey @KenanY good point. i think you are spot on. i was trying to keep it aligned with `npm install` as well as short. but i like "Install dependencies and then run tests". updating the PR now.

**Comment by ashleygwilliams** [comment]
@othiym23 @isaacs r?

File: doc/cli/npm-install-test.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    npm install-test <tarball file>
+    npm install-test <tarball url>
+    npm install-test <folder>
+
+    alias: npm it
+    common options: [--save|--save-dev|--save-optional] [--save-exact] [--dry-run]
+
+## DESCRIPTION
+
+    This command runs an `npm install` followed immediately by an `npm test`. It
```
Comment (iarna) [inline doc/cli/npm-install-test.md]: Indenting is making those whole section show as preformatted when that probably isn't what you wanted.

**Comment by othiym23** [comment]
Looks great! All we need now is for someone to write a test so we can land this and get it out! Thanks for putting this together, @ashleygwilliams!

**Comment by ashleygwilliams** [comment]
cool @othiym23 wanna merge this into that PR then?

**Comment by ashleygwilliams** [comment]
ooh but first lemme squash

**Comment by ashleygwilliams** [comment]
o god. this is what happens when you rebase before coffee.

**Comment by ashleygwilliams** [comment]
lol closing and opening a new on cuz i suck

**Pull request closed without merging**